modules/removing-egress-router-pods-before-initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="removing-egress-router-pods-before-initiating-limited-live-migration_{context}"]
+= Removing egress router pods before initiating the limited live migration
+
+Before initiating the limited live migration, you must check for, and remove, any egress router pods. If there is an egress router pod on the cluster when performing a limited live migration, the Network Operator blocks the migration and returns the following error: 
+
+[source,text]
+----
+The cluster configuration is invalid (network type live migration is not supported for pods with `pod.network.openshift.io/assign-macvlan` annotation. 
+
+Please remove all egress router pods). Use `oc edit network.config.openshift.io cluster` to fix.
+----
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Enter the following command to locate egress router pods on your cluster:
```

Comment:
Rewrite as: "To locate egress router pods on your cluster, enter the following command:"